app/src/lib/stores/app-store.ts

```diff
@@ -406,7 +402,7 @@ export class AppStore extends TypedBaseStore<IAppState> {
   private commitSummaryWidth = constrain(defaultCommitSummaryWidth)
   private stashedFilesWidth = constrain(defaultStashedFilesWidth)
 
-  private windowState: WindowState
+  private windowState: WindowState = 'normal'
```

Comment:
Should we initialize this to `null` or `undefined` instead? Checking the usages of this variable, I didn't see how setting 'normal' here could ever be a problem. However, I think using `null` or `undefined` would be more honest in case it actually matters.